src/services/db/review.ts

```diff
@@ -62,6 +64,22 @@ export const approvePullRequest = (
   siteName: string,
   pullRequestNumber: number
 ) =>
-  axiosInstance.post<void>(`${siteName}/pulls/${pullRequestNumber}/reviews`, {
-    event: "APPROVE",
-  })
+  axiosInstance.post<void>(
+    `${siteName}/pulls/${pullRequestNumber}/reviews`,
+    {
+      event: "APPROVE",
+    },
+    {
+      headers: {
+        // NOTE: This is currently done because
+        // we have a lock on the master branch
+        // and github requires an approval from
+        // *another* account that is not the creator
+        // of the pull request.
+        // This is a temporary workaround until we
+        // write a migration script to remove the lock on master.
+        // TODO!: Remove this
+        Authorization: `token ${E2E_TEST_GH_TOKEN}`,
```

Comment:
no strong opinion (@alexanderleegs thoughts?) - i went with this route but alex suggested doing migration script so that we can eliminate the need for extra tokens. 

i think our assumption is that the users aren't tech savvy enough to use the repo themselves

Reply:
I think we could set it up such that admin accounts can bypass the restrictions? We probably need to look into it later on, but for now i think this is fine